com.github.kevinsmia1939.PySimpleCV.yaml

```diff
@@ -0,0 +1,98 @@
+app-id: com.github.kevinsmia1939.PySimpleCV
```

Comment:
it should be `io.github`, this is now strictly enforced.

Reply:
Is that ok, there is no io website for this project.